k8sfv/pod.go

```diff
@@ -123,8 +125,21 @@ func createPod(clientset *kubernetes.Clientset, d deployment, nsName string, spe
 		panicIfError(err)
 		err = netlink.LinkSetNsFd(podIf, int(podNamespace.Fd()))
 		panicIfError(err)
-		err = podNamespace.Do(func(_ ns.NetNS) error {
-			return netlink.LinkSetUp(podIf)
+		err = podNamespace.Do(func(_ ns.NetNS) (err error) {
+			err = netlink.LinkSetUp(podIf)
+			if err != nil {
+				return
+			}
+			err = runCommand("ip", "a", "a", ip+"/32", "dev", veth.PeerName)
```

Comment:
I'd prefer if you'd spell out the `"ip", "addr", "add"`.  _I_ know what it means because I'm familiar with that command but it'll be hard to parse for a newbie.